### PR TITLE
Oauth SignIn feature implemented (updated)

### LINKED ISSUE #618: Add a way for sign in to game and rate it.
### Expected results
Add a way for sign in to game and rate it.  If not lot of points can add for one player.

### Actual results
Lot of points(ratings) can add for one player. 

### Steps to reproduce the issue
Add a way for sign in to game

### Implementation/Solution Ideas
Use gmail account instead or add a way for make a new account for game.

### Would you like to work on this
Open to anyone!

### PR DESCRIPTION
**Pull request by ShubhamChaudharyy**
Used the API token for Google Oauth feature.

- [x] Read and understood [see CONTRIBUTING.md]
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].

### Changes done in this Pull Request

- Fixes #618 

ezgif com-video-to-gif

### CODE REVIEW

**Comment by ShubhamChaudharyy** [comment]
@YashKumarVerma please review

File: js/Oauth.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+function onSignIn()
+ { console.log('hey')
```
Comment (YashKumarVerma) [inline js/Oauth.js]: :confused:

**Comment by YashKumarVerma** [comment]
# :tada: Good Work

File: js/Oauth.js (hunk 1 above)
Comment (YashKumarVerma) [reply js/Oauth.js]: Discussed in #1062

**Pull request merged**